probe-rs/src/architecture/arm/sequences/nrf.rs

```diff
@@ -53,10 +57,20 @@ fn unlock_core(
 
 /// Sets the network core to active running.
 fn set_network_core_running(interface: &mut dyn ArmProbe) -> Result<(), ArmError> {
-    interface.write_32(
-        APPLICATION_RESET_S_NETWORK_FORCEOFF_REGISTER as u64,
-        &[RELEASE_FORCEOFF],
-    )?;
+    // Determine if the RESET peripheral is mapped to secure or non-secure address space.
+    let periph_config_address = APPLICATION_SPU_PERIPH_PERM + 0x4 * APPLICATION_RESET_PERIPH_ID;
+    let periph_config = interface.read_word_32(periph_config_address)?;
```

Comment:
Can we have a proper bitfield for this?

Reply:
I think it's a bit overkill for just a single use.

Reply:
We can make one with just that one field :) I like it more, but we can also leave it as is. Feel free to do however :)